app/models/in_progress_form.rb

```diff
@@ -91,8 +92,45 @@ def expires_after
       end
   end
 
+  def log_cfi_metric(params)
+    return unless form_id == '21-526EZ'
+
+    # if the cfiMetric is present in the form already, then the disability education has been seen
+    if metadata['cfiMetric'].present?
+      params[:metadata]['cfiMetric'] = true
+      return
+    end
+
+    fd = params[:form_data] || params[:form_Data]
+    fd = JSON.parse(fd) if fd.is_a?(String)
+
+    return if fd.blank?
+
+    unless fd.dig('view:claim_type', 'view:claiming_increase') || fd.dig('view:claimType', 'view:claimingIncrease')
+      return
+    end
```

Comment:
Not sure if rubocop or reek have an opinion, but this can just be another `return unless`.

Reply:
rubocop. too long for return unless 🙄

Reply:
Ugh. It should've suggested a line break then, rather than rephrasing it. I'm not gonna fight this one 😛